src/pages/index.js

```diff
@@ -107,18 +134,86 @@ profileAddButton.addEventListener("click", function () {
 
 /* Functions */
 
+function handleAvatarUpdate({ link }) {
+  return api
+    .updateAvatar({ avatar: link })
+    .then((res) => {
+      userInfo.setUserInfo({
+        title: res.name,
+        description: res.about,
+        avatar: res.avatar,
+      });
+      setAvatarPopup.close();
```

Comment:
You close the modal inside `PopupWithForm`, so you don't need to duplicate it in `index.js`


**Please, correct it everywhere in the code**